src/main/java/net/gigaclub/buildersystemplugin/Commands/Worlds.java

```diff
@@ -244,14 +212,14 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 
                             JSONObject user = users.getJSONObject(j);
 
-                            Player player11 = Bukkit.getPlayer(user.getString("name"));
-                            String player21 = player11.toString();
+                            UUID uuid = UUID.fromString(user.getString("mc_uuid"));
+                            OfflinePlayer player11 = Bukkit.getOfflinePlayer(uuid);
+                            String player21 = player11.getName();
                             StringBuilder res1 = new StringBuilder();
```

Comment:
@Feier68 do not use numbers in variables. If you have duplicate variable names, assign the variable above and name the variables correctly.